Modules/rat_rescue.py

```diff
@@ -784,22 +787,33 @@ def rats(self, value):
                      await RatCache().get_rat_by_name(name))
             if found[0]:
                 self.rats.append(found[0])
-                return
+                found_rat = found[0]
             elif found[1]:
                 # a generic match (not platform specific) was found
                 # TODO throw a warning so the invoking method can handle this condition
                 log.warning("A match was found, but it was not the right platform!")
                 self.rats.append(found[1])
-                return
+                found_rat = found[1]
 
             else:
                 # lets make a new Rat!
                 if self.rat_board:  # PRAGMA: NOCOVER
```

Comment:
We may want to comment this whole logical branch out, as it doesn't serve a purpose until we have the API to develop against (which i hope is soon)

Reply:
Although this check may never be hit, I'd prefer leaving it there. I even wrote a test for it >.>